components/root/root.jsx

```diff
@@ -181,10 +180,6 @@ export default class Root extends React.PureComponent {
     }
 
     onConfigLoaded = () => {
-        if (isDevMode()) {
-            enableDevModeFeatures();
```

Comment:
This added a warning when trying to access a non-existent `.length` property of a Set or Map which lodash does in `isEqual` to try to identify the type of object. Since we have TypeScript now that will warn us if we do that by mistake, that warning isn't needed any more anyway.